homeassistant/components/backup/websocket.py

```diff
@@ -69,3 +74,47 @@ async def handle_create(
     manager: BackupManager = hass.data[DOMAIN]
     backup = await manager.generate_backup()
     connection.send_result(msg["id"], backup)
+
+
+@websocket_api.ws_require_user(only_supervisor=True)
+@websocket_api.websocket_command({vol.Required("type"): "backup/start"})
+@websocket_api.async_response
+async def handle_backup_start(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Backup start notification."""
+    manager: BackupManager = hass.data[DOMAIN]
+    manager.backing_up = True
+    LOGGER.info("Backup start notification")
```

Comment:
Is this information for the user or for developer/troubleshooting? If it's just the latter, please use debug level.

Reply:
TBH I do not know who this benefits, It was lifted from the current implementation.
debug make sense to me